tests/data/example_ode/config.toml

```diff
@@ -1,15 +1,14 @@
 name = "test-ode"
-kinetic_model = "kinetic_model.toml"
-priors = "priors.csv"
-measurements = "experiments.csv"
-biological_config = "biological_config.toml"
+kinetic_model_file = "kinetic_model.toml"
```

Comment:
IMO, I think that the "_file" suffix makes it less readable, but it is fine like that too.